### PR TITLE
Added new locations and images

### PR DESCRIPTION
**Pull request by SamuelMarks**
BTW: Got them from API calls

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@SamuelMarks__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

File: dokku-vm/azuredeploy.json [hunk 1]
```diff
@@ -69,17 +100,16 @@
         },
         "allowedValues": [
             "East US",
-            "East US 2",
-            "East Asia",
-            "West US",
```
Comment (singhkay) [inline dokku-vm/azuredeploy.json]: Looks like you removed a few locations such as East US 2, West US

Comment (SamuelMarks) [inline dokku-vm/azuredeploy.json]: Is that incorrect? - I literally posted the output of my API responses verbatim.

Comment (singhkay) [inline dokku-vm/azuredeploy.json]: What API are you calling. Here's what I see in this PR 
```
[
            "East US",
            "South Central US",
            "Central US",
            "North Europe",
            "West Europe",
            "Southeast Asia",
            "Japan West",
            "Japan East",
            "Australia Southeast",
            "Australia East"
       ]
```

Try calling this API - https://msdn.microsoft.com/en-us/library/azure/dn790540.aspx
`https://management.azure.com/subscriptions/<subscription-id>/locations?api-version={api-version}`

Comment (SamuelMarks) [inline dokku-vm/azuredeploy.json]: Weird. Anyway, running the official CLI client—with `azure location list --json`—gave me a bigger list which I committed up.

**Comment by singhkay** [comment]
lgtm, merged

**Pull request merged**